.ci/test_r_package_windows.ps1

```diff
@@ -95,6 +113,10 @@ Write-Output "Installing Rtools"
 ./Rtools.exe /VERYSILENT /SUPPRESSMSGBOXES /DIR=$RTOOLS_INSTALL_PATH ; Check-Output $?
 Write-Output "Done installing Rtools"
 
+# wait for all Rtools files to be written
+Write-Output "Sleeping to allow Rtools install to finish"
+Start-Sleep -Seconds 60
```

Comment:
Maybe we can use `Start-Process -FilePath Rtools.exe -NoNewWindow -Wait -ArgumentList ...` above to not wait manually?

Reply:
Oops! Seems I missed that:
> I hit "refresh" in Windows Explorer several times and did not see mingw_64 show up until after 20 seconds or so had passed. So I think the install returned control to powershell before all the files were actually written, somehow 😱 .

Indeed, very interesting observation! 👍 

Given that, please ignore my comment above about `-Wait` argument (it will not help). Instead, I believe we should do something like 
```
while True:
    sleep(1)
    if exists("$RTOOLS_INSTALL_PATH/mingw_64"):
        break
```
instead of hardcoded `sleep(60)`.

Reply:
:scream: I didn't know about `-Wait`! I don't write Powershell code anywhere else except this one script in this project, so I'm not very knowledgeable about it. Yes that seems a lot better than a sleep! Let me try that.

Reply:
Or you think that writing all files into `mingw_64` takes some significant time and this is not just about delay in `mingw_64` folder creation?

Reply:
ha we posted our responses at about the same time, I didn't see your second comment.

I don't think this `while` loop approach is exactly correct. The directory `mingw_64` could exist without all of the other files and folders in it being populated yet. This is what I meant by this explanation in the PR description

>  know fixing a problem by just introducing a sleep isn't ideal, but I think it would be difficult to figure out the full list of files needed by ld.exe when compiling LightGBM

Reply:
OK, I see... Could then please try `-Wait` workaround in RDP before merging this? I'm not an expert in PowerShell too, so maybe some magic will happen 😄

Reply:
sure! I won't be able to test it until later tonight though, maybe 6-8 hours from now

Reply:
No problem!
Maybe this is indeed a "recommended" official way of installation:
https://github.com/r-windows/rtools-base/blob/e60c36de70de7dedce89577c11d2fbe6008b2df4/scripts.ps1#L39

Reply:
oh nice, I forgot about `r-windows/r-tools`. I vaguely remember reading a blog post about it once. If that's how Jeroen does it, it's definitely a good sign.

Happy to say that switching to a `-Wait` did work! I could see that the Rtools install command didn't return control to powershell until both `ming_32` and `mingw_64` directories were created.

![image](https://user-images.githubusercontent.com/7608904/110418356-09d1f280-805d-11eb-9acf-c87fcb5fa67d.png)

![image](https://user-images.githubusercontent.com/7608904/110418371-10606a00-805d-11eb-8c3f-a494b32d4781.png)